Tasks/AzureResourceGroupDeploymentV2/package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "@types/node": "^10.17.0",
     "azure-pipelines-task-lib": "^3.0.6-preview.0",
-    "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.1",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.5",
```

Comment:
why is this change needed? `^` will take care of it. Or is it because of package-lock.json file?

Reply:
^ will take care of it, but just to be sure we will get the latest package.